### DISCUSSION TITLE
Site redirection with Japanese URL path

### DISCUSSION THREAD

**Discussion by hidech** [webstudio-is/webstudio-community#138] [Q&A] [answered]
Hi, How can I redirect a specific page to another?
When someone search our site on google, it shows old site build with another tool. The page no longer exists, so I want it to be redirected.
But Webstudio / Project settings / Redirects won't accept Japanese URL path like: /product-page/商品名-7
How can I resolve it?
<img width="633" alt="image" src="https://github.com/webstudio-is/webstudio-community/assets/35732672/60554b1f-f5da-46b2-97b3-dc447a69ea4e">

**Comment by kof** [comment]
Seems like our validation doesn't support non latin characters in the url right now

**Comment by kof** [comment, marked as answer]
follow this isue for updates https://github.com/webstudio-is/webstudio/issues/3225

**Reply by hidech** [reply]
Thanks, hope it will not be bothered.
